### PR TITLE
move mypy.ini and pytest.ini to setup.cfg

### PR DESCRIPTION
**Pull request by basnijholt**
Reduces the number of files in the root of the repo.

### CODE REVIEW

**Comment by jenshnielsen** [comment]
I have not been following the development of this part of the packaging that closely but are things not supposed to mov to pyproject.toml eventually?

**Comment by astafan8** [comment]
latest i know (been questing this qute recently) is that setup.cfg should be used only for super simple/trivial declarations. everything is indeed moving to toml, pytest since version 6 supports is completely. mypy still doesn't support toml properly so with that we should wait (or merge it into setup.cfg for now).

**Comment by astafan8** [comment]
on the other hand, if this just works, then it's fine. at later point we'll just migrate from single file setup.cfg to pyproject.toml and that's it

File: setup.cfg [hunk 1]
```diff
@@ -1,3 +1,73 @@
+[pytest]
```
Comment (jenshnielsen) [inline setup.cfg:1]: should this not be `[tool:pytest]` https://docs.pytest.org/en/stable/customize.html#setup-cfg as below

Comment (basnijholt) [reply setup.cfg]: You are correct, I've fixed it.

Comment (jenshnielsen) [reply setup.cfg]: There isanother section `[tool:pytest]` below

**Review by jenshnielsen — APPROVED** [review]

**Review by astafan8 — APPROVED** [review]

**Pull request merged**